static/app/app/controller.js

```diff
@@ -28,10 +28,11 @@ var Controller = Marionette.Object.extend({
         var mainRegion = channel.request('get:region', 'main');
         mainRegion.show(new LoadingView());
         var moves = channel.request('entities:moves');
-        $.when(moves.fetch(), spots.fetch()).done(function(){
+        $.when(saveAlert, moves.fetch(), spots.fetch()).done(function(saveAlert){
```

Comment:
you don't need to pass in `saveAlert` here. the arguments to this `$.when` statement should all be promises (which is what is returned by `fetch on a Backbone objet`. The idea is `when` it calls the callback when all the promises are resolved.